menas/ui/components/schema/schemaMain.controller.js

```diff
@@ -104,14 +104,17 @@ sap.ui.controller("components.schema.schemaMain", {
 	},
 
 	resetNewSchemaValueState : function() {
-		sap.ui.getCore().byId("newSchemaName").setValueState(sap.ui.core.ValueState.None)
-		sap.ui.getCore().byId("newSchemaName").setValueStateText("")
+		sap.ui.getCore().byId("newSchemaName").setValueState(sap.ui.core.ValueState.None);
+		sap.ui.getCore().byId("newSchemaName").setValueStateText("");
+
+    this.byId("fileUploader").setValueState(sap.ui.core.ValueState.None);
+    this.byId("fileUploader").setValueStateText("");
 	},
 
 	validateNewSchema : function() {
 		this.resetNewSchemaValueState();
 		var schema = this._model.getProperty("/newSchema")
-    var isOk = true
+    var isOk = true;
```

Comment:
I see there's a mix of tabs and spaces used for indentation. Please unify.
Also perhaps it brings up the question if to use tabs or spaces for `Javascript` (seems we have an agreement on spaces for `Scala`)

Reply:
I didnt touch the indentation of of the validateNewschema i added only the semicolon, but let me fix that as well.

Reply:
Thanks! 👍

Reply:
fixed.